lib/Makefile.am

```diff
@@ -78,11 +78,9 @@ tests_testRFC1738_LDADD= \
 tests_testRFC1738_LDFLAGS = $(LIBADD_DL)
 
 
-## Special Universal .h dependency test script
-## aborts if error encountered
-testHeaders: $(top_srcdir)/include/*.h
-	$(SHELL) $(top_srcdir)/test-suite/testheaders.sh "$(CXXCOMPILE)" $^ || exit 1
+## SOURCES cannot be used here
```

Comment:
Here and elsewhere, please give the reader at least some clue _why_ "SOURCES cannot be used". For example, here, they seem to be usable to me. They _currently_ do not contain any headers, but that current state should be irrelevant, right?

Reply:
We need to reference the headers from another directory that doesn't have its own Makefile.am. Clarifying.

Reply:
Would not it be better to add include/Makefile.am (than work around its absence and explain ourselves)? Or, even better, move include/ files into lib/?

Reply:
I'm reluctant to do that, it would violate longstanding tradition and I don't feel the gain is worth it

Reply:
I do not insist on this refactoring, although your "longstanding tradition" is a "longstanding annoyance" for some of us :-).